examples/meta/src/evaluation/multiclassovrevaluation.sg.in

```diff
@@ -0,0 +1,29 @@
+File traindat = csv_file("@SHOGUN_DATA@/fm_train_real.dat")
+File label_traindat = csv_file("@SHOGUN_DATA@/label_train_multiclass.dat")
+
+#![create_features]
+Features feats_train = features(traindat)
+MulticlassLabels ground_truth_labels = labels(label_traindat)
+#![create_features]
+
+#![create_classifier]
+Machine svm= machine("MulticlassLibLinear", C=1.0, labels=ground_truth_labels, seed=1)
+Parallel p = svm.get_global_parallel()
+p.set_num_threads(1)
+#![create_classifier]
+
+#![train_and_apply]
+svm.train(feats_train)
+Labels predicted_labels = svm.apply()
+RealVector labels = predicted_labels.get_real_vector("labels")
+#![train_and_apply]
+
+#![evaluate_accuracy]
+Evaluation binary_evaluator = evaluation("ROCEvaluation")
+Evaluation evaluator = evaluation("MulticlassOVREvaluation", binary_evaluation=binary_evaluator)
+mean_roc = evaluator.evaluate(predicted_labels,ground_truth_labels)
+
+Evaluation binary_evaluator = evaluation("ContingencyTableEvaluation", type="ACCURACY")
+Evaluation evaluator = evaluation("MulticlassOVREvaluation", binary_evaluation=binary_evaluator)
+mean_accuracy = evaluator.evaluate(predicted_labels,ground_truth_labels)
```

Comment:
Just to add to what Gil said, you will need to add also the variable type here or you will face the same error.

Reply:
Ahh got it! building now